### PR TITLE
Ky chok

### PR DESCRIPTION
**Pull request by kysnsoft**
first request

### CODE REVIEW

File: src/pages/ChokKahYang/landingKY.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        this.state = {
+            name: "KY Chok"
+
+        };
+
+    }
+
+    render() {
+        return (
+            <BaseKY title={"Chok Kah Yang (KY)"}>
```
Comment (yuhosns) [inline src/pages/ChokKahYang/landingKY.jsx]: good try but please reuse component (Basepage in this case) if possible. Less code = less maintain, less file size

File: src/pages/ChokKahYang/baseKY.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import "./base.scss"
+import EduList from "./eduList"
+import Profile from "./profile.jpg"
+
+export default class BaseKY extends React.Component {
```
Comment (yuhosns) [inline src/pages/ChokKahYang/baseKY.jsx]: move these content to your landingky and remove this component

Comment (ChanVinShengSNsoft) [reply src/pages/ChokKahYang/baseKY.jsx]: Ky chok absent today

File: src/pages/ChokKahYang/eduList.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+
+        const { school, qualified, yearStart, yearEnd } = this.props.qualification
+
+        return (
+            <div className="containerEdu">
```
Comment (yuhosns) [inline src/pages/ChokKahYang/eduList.jsx]: element class name should be kebab-case, but i will ignore this time

**Comment by yuhosns** [comment]
gd

File: src/pages/ChokKahYang/eduList.jsx (hunk 3 above)
Comment (kysnsoft) [reply src/pages/ChokKahYang/eduList.jsx]: Noted with thanks.
so for this example will be container-edu?

Comment (yuhosns) [reply src/pages/ChokKahYang/eduList.jsx]: yeap

**Pull request merged**